src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random rm = new Random();
+
+    public int getRmInt(int rmInt) {
+        return rm.nextInt(rmInt) + 1;
+    }
+
     public String getRandomColor() {
-        return null;
+        Colors[] values = Colors.values();
+        int length = values.length;
+        int randIndex = rm.nextInt(length);
+        Colors colors = values[randIndex];
+        return colors.toString();
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants